runtime/src/constants.rs

```diff
@@ -1,13 +1,13 @@
 pub mod currency {
 	use subsocial_primitives::Balance;
 
-	pub const UNITS: Balance = 100_000_000_000;
-	pub const DOLLARS: Balance = UNITS;            // 100_000_000_000
-	pub const CENTS: Balance = DOLLARS / 100;      // 1_000_000_000
-	pub const MILLICENTS: Balance = CENTS / 1_000; // 1_000_000
+	pub const UNITS: Balance = 10_000_000_000;
+	pub const DOLLARS: Balance = UNITS;            // 10_000_000_000
+	pub const CENTS: Balance = DOLLARS / 100;      // 100_000_000
+	pub const MILLICENTS: Balance = CENTS / 1_000; // 100_000
 
 	pub const fn deposit(items: u32, bytes: u32) -> Balance {
-		items as Balance * 15 * CENTS + (bytes as Balance) * 6 * CENTS
+		items as Balance * 20 * DOLLARS + (bytes as Balance) * 100 * MILLICENTS
```

Comment:
why this change?

Reply:
We don't use `deposit` function for now, but I aligned this calculations with https://github.com/paritytech/polkadot/blob/16e6ce0ca0330aad1e0beea2b064f704ce92f207/runtime/polkadot/constants/src/lib.rs#L36

Reply:
Okay then, good job ❤️